eradiate/_presolver.py

```diff
@@ -38,10 +38,17 @@ def reset(self):
         ``[$PWD, $ERADIATE_DIR/resources/data]``
         """
 
+        eradiate_dir = os.getenv("ERADIATE_DIR")
+        if eradiate_dir is None:
+            raise ValueError(f"Environment variable ERADIATE_DIR is not set.\n"
+                           + f"Please set this variable to the absolute path of the Eradiate installation folder.\n"
+                           + f"If you are running Eradiate directly from the sources, " + 
+                           "then you can alternatively source the provided setpath.sh script.")
```

Comment:
No need for an f-string here; in addition, it's a good habit to avoid concatenating strings if it's just about breaking a long one during definition.

```suggestion
            raise ValueError(
                "Environment variable ERADIATE_DIR is not set.\n"
                "Please set this variable to the absolute path of the Eradiate installation folder.\n"
                "If you are running Eradiate directly from the sources, " 
                "then you can alternatively source the provided setpath.sh script."
            )
```

Reply:
Hum you're absolutely right, I will improve this format

I will also check the existence of this file

Reply:
Done